golang/1.13/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.13-alpine3.11
+
+RUN apk add git make --update --no-cache \
+    && go get -u golang.org/x/lint/golint \
```

Comment:
The error message comes from the fact that an `&&` operator is missing from the end of this line.